tests/Neo.VM.Tests/Tests/Others/Init.json

```diff
@@ -0,0 +1,41 @@
+{
+  "category": "Others",
+  "name": "Init",
+  "tests": [
+    {
+      "name": "Init script",
```

Comment:
I've just realized that we'll have to import whole neo-project/neo as a submodule to get these tests now...

https://github.com/nspcc-dev/neo-go/tree/de2a44508893db9ea60a08bf88c293488feed333/pkg/vm/testdata

Reply:
Hi @roman-khimov ,  What exactly is being used from this repository? 
I've seen a few JSON files with some tests. Do you use those JSON files or do you need something else?

Reply:
https://github.com/nspcc-dev/neo-go/blob/de2a44508893db9ea60a08bf88c293488feed333/pkg/vm/json_test.go#L94

We run these tests for NeoGo implementation of VM.